tools/modules/EnsEMBL/Web/Component/Tools/VEP/InputForm.pm

```diff
@@ -396,13 +396,13 @@ sub _build_identifiers {
   });
 
   $fieldset->add_field({
-    'field_class' => '_stt_core _stt_merged _stt_gencode_basic',
+    'field_class' => '_stt_Homo_sapiens _stt_Mus_musculus _stt_core _stt_merged _stt_gencode_basic',
     'type'        => 'checkbox',
     'name'        => 'ccds',
     'label'       => $fd->{ccds}->{label},
     'helptip'     => $fd->{ccds}->{helptip},
     'value'       => 'yes',
-  });
+  }) if (first { $_->{'value'} eq 'Homo_sapiens' } @$species);
```

Comment:
Missed Mus_musculus?

Reply:
Fall in the same category as vertebrates so we only need to check for human or mouse, both is not needed

Reply:
wonder why did it have _stt_mus_musulus mentioned in the class before?

Reply:
in the class it is the correct way because it has to toggle between mouse and human but the if check in the perl is to add them to the form first. Remember the fix we discuss to generate the _stt toggle data for each species.